bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js

```diff
@@ -0,0 +1,121 @@
+import Presentations from '/imports/api/2.0/presentations';
+import Auth from '/imports/ui/services/auth';
+import { makeCall } from '/imports/ui/services/api';
+
+const CONVERSION_TIMEOUT = 300000;
```

Comment:
This should be in the config, in my opinion.

Reply:
Will leave this change for the next iteration.

Reply:
Add this to the list of issues?